transformations/multilingual_lexicon_perturbation/README.md

```diff
@@ -0,0 +1,69 @@
+Northern Sotho (ns), Occitan (post 1500) (oc), Oriya (or), Panjabi; Punjabi (pa), Polish (pl), Pushto; Pashto (ps), Portuguese (pt),
+Romanian; Moldavian; Moldovan (ro), Russian (ru), Sindhi (sd), Sinhala; Sinhalese (si), Slovak (sk), Slovenian (sl), Somali (so), Albanian (sq),
+Serbian (sr), Swati (ss), Sundanese (su), Swedish (sv), Swahili (sw), Tamil (ta), Thai (th), Tagalog (tl), Tswana (tn), Turkish (tr),
+Ukrainian (uk), Urdu (ur), Uzbek (uz), Vietnamese (vi), Wolof (wo), Xhosa (xh), Yiddish (yi), Yoruba (yo), Chinese (zh), Zulu (zu)
+
+The multilingual lexicon is generated by using the M2M100 model, Beyond English-Centric Multilingual Machine Translation (JMLR 2021). 
+
+Processing speed of both Multilingual Lexicon Perturbation vs [Mixed Language Perturbation](https://github.com/gentaiscool/NL-Augmenter/tree/mixed_language_perturbation/transformations/mixed_language_perturbation) tested on the 36 sentences in the `test.json` is 160x faster on the lexicon based one. The comparison of both perturbation are shown below:
+<h1 align="center">
+  <img src="img/speed_comparison.jpeg"/>
+</h1>
+
+Author names:
+- Genta Indra Winata (giwinata@connect.ust.hk, The Hong Kong University of Science and Technology),
+- Samuel Cahyawijaya (scahyawijaya@connect.ust.hk, The Hong Kong University of Science and Technology)
+- Bryan Wilie (bryanwilie92@gmail.com, Institut Teknologi Bandung).
+
+## What type of a transformation is this?
+This transformation acts as a perturbation to test robustness. Few words were picked at random with a probability and translated to the target language if the word is in the lexicon dctionary.
+
+## Previous Work
+(1) Mixed-Language Training (Published in AAAI 2020)
+```bibtex
+@inproceedings{liu2020attention,
+  title={Attention-informed mixed-language training for zero-shot cross-lingual task-oriented dialogue systems},
+  author={Liu, Zihan and Winata, Genta Indra and Lin, Zhaojiang and Xu, Peng and Fung, Pascale},
+  booktitle={Proceedings of the AAAI Conference on Artificial Intelligence},
+  volume={34},
+  number={05},
+  pages={8433--8440},
+  year={2020}
+}
+```
+(2) Continual Mixed-Language Pre-Training (Accepted in ACL Findings 2021)
+```
+@article{liu2021continual,
+  title={Continual Mixed-Language Pre-Training for Extremely Low-Resource Neural Machine Translation},
+  author={Liu, Zihan and Winata, Genta Indra and Fung, Pascale},
+  journal={arXiv preprint arXiv:2105.03953},
+  year={2021}
+}
+```
+(3) M2M100 Model, Beyond English-Centric Multilingual Machine Translation (JMLR 2021)
+```
+@article{fan2021beyond,
+  title={Beyond english-centric multilingual machine translation},
+  author={Fan, Angela and Bhosale, Shruti and Schwenk, Holger and Ma, Zhiyi and El-Kishky, Ahmed and Goyal, Siddharth and Baines, Mandeep and Celebi, Onur and Wenzek, Guillaume and Chaudhary, Vishrav and others},
+  journal={Journal of Machine Learning Research},
+  volume={22},
+  number={107},
+  pages={1--48},
+  year={2021}
+}
+
```

Comment:
Also, please add a Data and Code Provenance section which describes where the data comes from.

Reply:
Is this "Data and Code Provenance" section acceptable? or should we add/modify anything from here?
https://github.com/GEM-benchmark/NL-Augmenter/blob/ba9531888469e3e8c4d7e687c15103a7e72cf23e/transformations/multilingual_lexicon_perturbation/README.md

Reply:
This should work!